### PR TITLE
Add Russian translation.

### PR DESCRIPTION
**Pull request by DimmKG**
Translated all strings into Russian, but some words are untranslatable, because the translation file doesn't provide strings for them.

For example: 
- True or False; 
- video, clip, stream, video qualities, clip qualities, and stream qualities in "Remove %s quality" dialogs;
- user, game, user blacklist and game blacklist in "Remove %s from blacklist" dialogs.

### CODE REVIEW

File: addon.xml [hunk 1]
```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.twitch" version="2.5.4" name="Twitch" provider-name="anxdpanic, A Talented Community">
+<addon id="plugin.video.twitch" version="2.5.4+matrix.1" name="Twitch" provider-name="anxdpanic, A Talented Community">
```
Comment (anxdpanic) [inline addon.xml]: Please remove this from the PR

File: addon.xml [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
     <requires>
-        <import addon="xbmc.python" version="2.20.0"/>
+        <import addon="xbmc.python" version="3.0.0"/>
```
Comment (anxdpanic) [inline addon.xml]: Please remove this from the PR

File: addon.xml [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
         <import addon="script.module.six" version="1.11.0"/>
         <import addon="script.module.python.twitch" version="2.0.16"/>
         <import addon="script.module.requests" version="2.9.1"/>
     </requires>
     <extension point="xbmc.python.pluginsource" library="resources/lib/addon_runner.py">
         <provides>video</provides>
     </extension>
-    <extension point="xbmc.service" library="resources/lib/service_runner.py" start="login"/>
+    <extension point="xbmc.service" library="resources/lib/service_runner.py"/>
```
Comment (anxdpanic) [inline addon.xml]: Please remove this from the PR

**Review by anxdpanic — CHANGES_REQUESTED** [review]
Thank you for the PR, and sorry for the delay. I have requested a few changes before this can be merged.

File: addon.xml (hunk 3 above)
Comment (DimmKG) [reply addon.xml]: I've done it. Looks like, It's incorrect fork. I did that without Git.

**Pull request merged**